quick-start/README.md

```diff
@@ -0,0 +1,169 @@
+# Quick Start
+
+The purpose of this walkthrough is to allow a quick test for this plugin.
+Some docker containers will be used to allow this test:
++ Nexus
++ Keycloak
++ Postgres
++ XWiki
+
+### Requirements
+
+Requirement  | Minimum version
+------------ | -------------
+Java    | >= 8
+Maven   | >= 3.8
+Docker  | >= 20
+Curl    | >= 7
+
+Run the following command inside the quick-start folder:
+
+```sh
+$ chmod +x setup.sh
+$ docker-compose up -d
+```
+
+## Nexus
+
++ Wait until nexus is ready, it may take a while.
++ URL: http://localhost:8081
++ User: admin
++ Password: get the password running the following command:
+    ```
+    docker-compose exec nexus cat /nexus-data/admin.password
+    ```
++ Add admin permission to anonymous user.
+    + Go to Administration menu, select User and click over anonymous user.
+        + Add nx-amin role to the annymous user.
```

Comment:
I believe many of the previous comments were not resolved and may have been missed due to the file move. 
Expand the hidden conversations and check those. These two lines have 2 comments that were not solved.

You may choose to ignore some, assuming the person is a proper dev and has more knowledge, but some are related to broken style or English, which you should not.